src/styles/App.scss

```diff
@@ -0,0 +1,25 @@
+html {
+  margin: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: #f2f2f2;
```

Comment:
Could this be a SCSS variable?

Reply:
I keep getting a build error when I try and use variables on this page.

Reply:
Do you have it imported at the top?